### ISSUE TITLE
Analytical derivatives of SOAP

### ISSUE THREAD

**Issue by VondrakMar** [SINGROUP/dscribe#95]
Hello,
I noticed that 
```
der,desc = soap.derivatives(mol,method="analytical",attach=True)
```
is not working (ValueError: Analytical derivatives not currently available when attach=True.).

May i ask if there is any plan of implementing attaching a SOAP to the displaced atom with analytical derivatives, please?

**Comment by lauri-codes** [comment]
Hi @VondrakMar,

Yes, the analytical derivatives are not currently available with this option. At the moment there are no plans for implementing them, but it may be possible if there are several requests for this. Currently, you can still get them by using `method="numerical" ` which will just be somewhat slower.

**Comment by Nilsgoe** [comment]
Besides the `method="analytical` also  `periodic = True` seems not be available for soap.derivatives. I would be interested in both options.

**Comment by lauri-codes** [comment]
@Nilsgoe: Would numerical SOAP derivatives for periodic systems be OK for you? If so, could you create a new issue and we can continue discussion there.

**Comment by lauri-codes** [comment]
`soap.derivatives(mol,method="analytical",attach=True)` now available in v2.0.0.
